modules/local/report.nf

```diff
@@ -130,14 +130,13 @@ def generate_coverage_data(sample_data, bp_field, species){
                 && species[species_data_pos].fixed_genome_size != null){
 
                 def length = species[species_data_pos].fixed_genome_size.toLong()
-                def cov = base_pairs / q_length
+                def cov = base_pairs / length
```

Comment:
What's the difference between `q_length` and `length`?

Reply:
Just verifying I have made this change correctly then I will update the comment. Kyla had the same concern.

Reply:
`q_length` or query length here is the genome length generated by quast in this case or whatever path is pointed to in the `QCReportFields` of the `nextflow.config` while the `length` field is the length specified in the `QCReport` section for a specific organism.

Reply:
Thanks so much for the explanation. So switching to `length` here means that the coverage value is calculated based on a fixed genome length rather than on the calculated length. And this is what is desired.

That makes sense, I just wanted to check/understand what was going on.